<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formWindow</class>
 <widget class="QDialog" name="formWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>331</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Body Temp (Â°C):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBodyTemp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BP Right Hand:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBloodLeft">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>BP Left Hand:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBloodRight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Heart Rate:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHeartRate">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Sleeping Time:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSleep">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Weight:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWeight">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="cBoxEmotion">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>101</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>:|</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>:)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>:(</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cBoxFeeling">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>111</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Good</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Meh</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bad</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtNotes">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnConfirm">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>300</y>
      <width>221</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
